provisioning/device/src/Authentication/AuthenticationProviderX509.cs

```diff
@@ -16,6 +16,13 @@ public class AuthenticationProviderX509 : AuthenticationProvider
         /// </summary>
         /// <remarks>
         /// Ensure that you dispose any supplied <see cref="X509Certificate2"/> after you are done using it to ensure there are no memory leaks.
+        /// <para>
+        /// The certificates provided are assumed to be a full chain and are not validated. The caller should
+        /// validate them independently.
+        /// </para>
+        /// <para>
+        /// If the certificates need to be installed to the local certificate store, that should be done by the user.
```

Comment:
Do the ms docs have any articles about this? If so it could be nice to link one.

Reply:
I don't know of any.